packages/cypress/cypress/integration/cucumber/step_definitions/refapp-3.x/05-vitals-and-biometrics/vitals-and-biometrics.js

```diff
@@ -0,0 +1,56 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+let identifier = null;
+let patient = null;
+
+before({ tags: "@vitals-and-biometrics" }, () => {
+  cy.generateIdentifier().then((generatedIdentifier) => {
+    identifier = generatedIdentifier;
+    cy.createPatient(identifier).then((generatedPatient) => {
+      patient = generatedPatient;
+      cy.startFacilityVisit(patient.uuid);
+    });
+  });
+});
+
+Given("the user is logged in", () => {
+  cy.login();
+});
+
+Given("the user arrives on a patient’s summary page", () => {
+  cy.visit(`patient/${patient.uuid}/chart`);
+});
+
+When("the user clicks on Record Vitals and Biometrics", () => {
+  cy.contains("Vitals & Biometrics").click({ force: true });
+  cy.contains("Record biometrics").click({force:true});
+});
+
+Then("the Vitals form should load", () => {
+  cy.contains("Record Vitals and Biometrics");
+});
+
+When("the user adds vitals", () => {
+  cy.get("#systolic").type("120", { force: true });
```

Comment:
Can we get elements by using their labels?

Reply:
In vitals record form no labels are used for the naming. So we have to use id instead of using labels.
ss-
<img width="846" alt="image" src="https://user-images.githubusercontent.com/77311602/188065610-3b45bb75-aa4e-4ea6-9dca-8d85ea88a65c.png">

Reply:
Alright!